dojo/tools/csaf/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Hochschule Furtwangen (Summer semester 2022) - Manuel Johannes Messner, Bastian Hodapp, Aljosha Vieth, Cornelius Schaub, Dominik Schwerle'
```

Comment:
```suggestion
```
We track authorship in the VCS. Please avoid this meta attributes in parser code.

Reply:
I've seen several parsers where people put some sort of username in, such as https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/dawnscanner/__init__.py

We can remove it, though :)